Atualização dos dados do calendário e inserção de um HTML para downlo…

Joao-Diogo-Figueirinha-Rodrigues opened the pull request:
…ad dos ficheiros

https://trello.com/c/oMdk30ps/6-representa%C3%A7%C3%A3o-do-calend%C3%A1rio-em-html

Joao-Diogo-Figueirinha-Rodrigues committed:
Atualização dos dados do calendário e inserção de um HTML para download dos ficheiros

https://trello.com/c/oMdk30ps/6-representa%C3%A7%C3%A3o-do-calend%C3%A1rio-em-html

Joao-Diogo-Figueirinha-Rodrigues committed:
Atualização dos dados do calendário e inserção de um HTML para download dos ficheiros ou para inserção de URL da sua localozação

https://trello.com/c/oMdk30ps/6-representa%C3%A7%C3%A3o-do-calend%C3%A1rio-em-html

jmmpo3 commented on src/main/resources/script/uploadDocs.html:
```diff
@@ -0,0 +1,59 @@
+      <div id="showFile">
+        <input type="url" name="url" placeholder="Insira URL">
+        <label for="url">Inserir URL</label>
+        <input type="submit" value = "Submeter" action="\calendar\consume\url">
```
O conteúdo da action deve ser colocado no tag form.

jmmpo3 commented on src/main/resources/script/uploadDocs.html:
```diff
@@ -0,0 +1,59 @@
+    radioFile.addEventListener("click", () => {
+      if (radioFile.checked) {
+        inputContainerFile.style.display = "block";
+        inputContainerUrl.style.display = "none";
```
Acrescentar: document.getElementById("formId").action = "\calendar\consume\file";

Joao-Diogo-Figueirinha-Rodrigues replied:
Alterado para formIdFile para distinção do id referente ao Url

jmmpo3 replied:
só tens um id="formId" que estará na tag de form.

jmmpo3 commented on src/main/resources/script/uploadDocs.html:
```diff
@@ -0,0 +1,59 @@
+    radioURL.addEventListener("click", () => {
+      if (radioURL.checked) {
+        inputContainerFile.style.display = "none";
+        inputContainerUrl.style.display = "block";
```
Acrescentar: document.getElementById("formId").action = "\calendar\consume\url";

Joao-Diogo-Figueirinha-Rodrigues replied:
Alterado para formIdUrl para destinção do id referente aos ficheiros

jmmpo3 commented on src/main/resources/script/uploadDocs.html:
```diff
@@ -0,0 +1,59 @@
+  </head>
+  <body>
+    <h1>Faça Upload do seu Ficheiro</h1>
+    <form class="" action="index.html" method="post"> <br>
```
id="formId"

jmmpo3 commented on src/main/resources/script/uploadDocs.html:
```diff
@@ -0,0 +1,59 @@
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Upload de Ficheiros</title>
```
Carregar novo calendário

jmmpo3 commented on src/main/resources/script/uploadDocs.html:
```diff
@@ -0,0 +1,59 @@
+    <title>Upload de Ficheiros</title>
+  </head>
+  <body>
+    <h1>Faça Upload do seu Ficheiro</h1>
```
Carregar novo calendário

jmmpo3 commented on src/main/resources/script/uploadDocs.html:
```diff
@@ -0,0 +1,59 @@
+    <h1>Faça Upload do seu Ficheiro</h1>
+    <form class="" action="index.html" method="post"> <br>
+
+      <p>Escolha a forma de fazer upload do ficheiro</p>
```
... a forma de carregar um novo calendário.

jmmpo3 commented on src/main/resources/script/uploadDocs.html:
```diff
@@ -0,0 +1,59 @@
+      </div>
+
+<br>
+    <input type="submit" value = "Submeter">
```
value="Carregar calendário"

jmmpo3 replied:
E só é necessário este submit, o outro pode ir à vida.

jmmpo3 commented on src/main/resources/script/uploadDocs.html:
```diff
@@ -0,0 +1,59 @@
+
+
+      <div id="hideFile">
+        <input type="file" name="upload" value="upload"> <br>
```
name="file"

jmmpo3 replied:
e a apagar o value!

jmmpo3 commented on src/main/resources/script/uploadDocs.html:
```diff
@@ -0,0 +1,59 @@
+      </div>
+
+      <div id="showFile">
+        <input type="url" name="url" placeholder="Insira URL">
```
Falta um input que é radio, que é o tipo de formato do ficheiro remoto, ou CSV ou JSON.

Joao-Diogo-Figueirinha-Rodrigues replied:
É necessário 2 radio buttons para diferenciar o ficheiro visto que a action será a mesma?

Joao-Diogo-Figueirinha-Rodrigues committed:
Atualização dos dados do calendário e inserção de um HTML para download dos ficheiros ou para inserção de URL da sua localização

https://trello.com/c/oMdk30ps/6-representa%C3%A7%C3%A3o-do-calend%C3%A1rio-em-html

Joao-Diogo-Figueirinha-Rodrigues committed:
Retificação de alguns erros

https://trello.com/c/oMdk30ps/6-representa%C3%A7%C3%A3o-do-calend%C3%A1rio-em-html

Joao-Diogo-Figueirinha-Rodrigues committed:
Criação de mais um radio Button e correção de outros erros
https://trello.com/c/oMdk30ps/6-representa%C3%A7%C3%A3o-do-calend%C3%A1rio-em-html

jmmpo3 commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Representação do calendário em HTML](https://trello.com/c/oMdk30ps/6-representa%C3%A7%C3%A3o-do-calend%C3%A1rio-em-html)
